README.md

```diff
@@ -1,4 +1,4 @@
-![Space Station 14](https://i.imgur.com/DKXu8kw.png)
+![Space Station 14](https://raw.githubusercontent.com/space-wizards/asset-dump/master/ss14.png)
```

Comment:
You should refer to these with a perma link to the commit (`3dd3078e49e3a7e06709a6e0fc6e3223d8d44ca2`) instead of to `master`